variables.tf

```diff
@@ -118,6 +118,12 @@ variable "ui_certificate_arn" {
   description = "SSL certificate for UI. If set to empty string, UI is disabled. "
 }
 
+variable "ui_allow_list" {
+  type        = list(string)
+  default     = []
+  description = "List of CIDRs we want to grant access to our Metaflow UI Service. Usually this is our VPN's CIDR blocks."
+}
```

Comment:
It looks like its a different default from what what ui module is using, we should probably make them the same. Does setting it to `[]` block all public access? If so maybe we should switch the other default to that too.

Reply:
You're right, the UI module defaulted to allowing all ingress. Setting `ui_allow_list` to `[]` blocks all ingress, which is typically a better default. I updated the default value in the UI module.

It's worth noting that this could impact users, since they might not be able to access the UI anymore if they were relying on the previous default. That being said, it's probably for the better because allowing all traffic by default is rarely a good idea.

Reply:
I agree, its a better default. I'll make a note in release notes